### PR TITLE
chore: add DATRON case study

### PR DESCRIPTION
**Pull request by bsmth**
__Description:__
* Adds a DATRON case study
* Improve quote section style on Toggle's case study
* wider jumbotron area for case study article title to appear on single line

### CODE REVIEW

File: src/pages/case-study/datron.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+            />
+            Low integration costs
+          </p>
+          <p className={ouCss.outcome}>
+            <img
+              alt="Workflow icon"
+              className={ouCss.outcome__icon}
+              src="/img/pages/case-study/icons/workflow.svg"
+            />
+            Wide compatiblility and simple configuration
```
Comment (the-sheev) [inline src/pages/case-study/datron.tsx:85]:
```suggestion
            Wide compatibility and simple configuration
```

File: src/pages/case-study/datron.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+            efficiently.
+          </p>
+          <p className="font-size--large">
+            Initially, DATRON was experimenting with InfluxDB as a datastore but
+            immediately encountered problems ingesting high-throughput
+            workloads. A recurring issue that surfaced was InfluxDB consuming
+            too much system RAM and an inability to keep up with ingestion
+            rates, causing frequent memory cache flushing. QuestDB was a capable
+            drop-in replacement in this scenario and allowed DATRON to do more
+            with less hardware, more efficiently.
```
Comment (the-sheev) [inline src/pages/case-study/datron.tsx:171]:
```suggestion
            with less hardware.
```
is more impressive and reads better.

File: src/pages/case-study/datron.tsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+          <h3>Simplifying the integration</h3>
+          <p className="font-size--large">
+            DATRON use the QuestDB Docker image, which is a turnkey,
+            out-of-the-box solution. Using ILP (InfluxDB Line Protocol), the
+            data is inserted and read out via REST API to other software for
+            analysis. Inserting and reading the information is possible with
+            popular high-level protocols and little programming effort, a
+            definite advantage over hand-crafted or bespoke solutions.
+            <img
```
Comment (the-sheev) [inline src/pages/case-study/datron.tsx:182]: `InfluxDB line Protocol` should be `InfluxDB line protocol`.

File: src/pages/case-study/datron.tsx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+          </p>
+
+          <h3>Feedback from a thriving community</h3>
+          <p className="font-size--large">
+            The developer and community support meant that questions get
+            answered quicker, leading to faster integration and troubleshooting
+            through shorter feedback loops. Using Slack as a communication tool
+            is an additional convenience that bridges the gap between ideas and
+            implementation and adds to the agility of the project to deliver a
+            feature addition or a bug fix.
```
Comment (the-sheev) [inline src/pages/case-study/datron.tsx:198]:
```suggestion
            feature addition or bug fix.
```

File: src/pages/case-study/datron.tsx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+              className={chCss.chart}
+              height={347}
+              src="/img/pages/case-study/datron/chart.png"
+              width={500}
+            />
+          </p>
+
+          <h3>Feedback from a thriving community</h3>
+          <p className="font-size--large">
+            The developer and community support meant that questions get
```
Comment (the-sheev) [inline src/pages/case-study/datron.tsx:193]:
```suggestion
            The developer and community support means that questions get
```

File: src/pages/case-study/toggle.tsx [hunk 6]
```diff
@@ -11,9 +11,9 @@ import ouCss from "../../css/case-study/outcome.module.css"
 import seCss from "../../css/section.module.css"
 
 const Toggle = () => {
-  const title = "Case Study: Toggle AI"
+  const title = "Toggle AI - time series data for machine learning in FinTech"
   const description =
-    "How Toggle switched from InfluxDB to QuestDB and benefited from massive cost reduction + performance improvements"
+    "Toggle switched from InfluxDB to QuestDB for and benefited from faster queries, massive cost reduction, and performance improvements on ingestion."
```
Comment (the-sheev) [inline src/pages/case-study/toggle.tsx:16]:
```suggestion
    "Toggle switched from InfluxDB to QuestDB and benefited from faster queries, massive cost reduction, and performance improvements on ingestion."
```

File: src/pages/customers.tsx [hunk 7]
```diff
@@ -150,7 +150,35 @@ const Customers = () => {
           />
         </Button>
       </section>
-
+      <section className={clsx(seCss.section, seCss["section--inner"])}>
+        <div className={caCss.card}>
+          <div className={caCss.card__illustration}>
+            <img
+              alt="TODO"
```
Comment (the-sheev) [inline src/pages/customers.tsx:157]: TODO

File: src/pages/case-study/datron.tsx [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+          <h3>Simplifying the integration</h3>
+          <p className="font-size--large">
+            DATRON use the QuestDB Docker image, which is a turnkey,
+            out-of-the-box solution. Using ILP (InfluxDB Line Protocol), the
+            data is inserted and read out via REST API to other software for
+            analysis. Inserting and reading the information is possible with
+            popular high-level protocols and little programming effort, a
+            definite advantage over hand-crafted or bespoke solutions.
+            <img
+              alt="Chart showing the average transaction duration for QuestDB on a given day"
```
Comment (the-sheev) [inline src/pages/case-study/datron.tsx:183]: Wrong.

File: src/pages/case-study/toggle.tsx [hunk 9]
```diff
@@ -11,9 +11,9 @@ import ouCss from "../../css/case-study/outcome.module.css"
 import seCss from "../../css/section.module.css"
 
 const Toggle = () => {
```
Comment (the-sheev) [inline src/pages/case-study/toggle.tsx:13]: Where are this file and the following file displayed? I don't see them when building the project.

**Review by the-sheev — CHANGES_REQUESTED** [review]

File: src/pages/case-study/datron.tsx (hunk 1 above)
Comment (bluestreak01) [inline src/pages/case-study/datron.tsx:85]:
```suggestion
            Wide compatibility and simple configuration
```

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: src/pages/case-study/toggle.tsx (hunk 9 above)
Comment (bsmth) [reply src/pages/case-study/toggle.tsx:13]: Which files? The case study pages are rendered to `/case-study/datron/` and `/case-study/toggle/`, does that answer the question?

File: src/pages/case-study/datron.tsx (hunk 3 above)
Comment (bsmth) [reply src/pages/case-study/datron.tsx:182]: Needs to be updated in the diagram also - can do as follow-up

**Review by the-sheev — APPROVED** [review]

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**